src/include/duckdb/parser/qualified_name.hpp

```diff
@@ -26,7 +26,7 @@ struct QualifiedName {
 		vector<string> entries;
 		string entry;
 	normal:
-		// quote
+		//! quote
```

Comment:
The exclamation marks should only be added to comments before functions or classes. This is relevant only for [doxygen](https://riptutorial.com/doxygen/example/27439/commenting-your-code) (although we haven't generated doxygen docs in forever).